app/controllers/queues_controller.rb

```diff
@@ -0,0 +1,60 @@
+class QueuesController < ApplicationController
+  def spam
+    @request = RescueRequest.find_by(needs_spam_check: nil)
+  end
+
+  def spam_spam
+    request = RescueRequest.find(params[:request_id])
```

Comment:
You've got an identical line in a few of these actions; you can shunt that off to a `before_action` callback.